README.md

```diff
@@ -33,6 +33,18 @@ It's as simple as:
 pip install taxpasta
 ```
 
+Taxpasta is also available from the [Bioconda](https://bioconda.github.io/)
+channel
+
+```shell
+conda install -c bioconda taxpasta
+```
+
+and thus automatically generated
+[Docker](https://quay.io/repository/biocontainers/taxpasta?tab=tags) and
+[Singularity](https://depot.galaxyproject.org/singularity/)
+[BioContainers](https://biocontainers.pro/) images also exist.
+
 ## Copyright
 
 -   Copyright © 2022, Moritz E. Beber, Maxime Borry, James A. Fellows Yates, and
```

Comment:
```suggestion
-   Copyright © 2023, Moritz E. Beber, Maxime Borry, James A. Fellows Yates, and
```

Reply:
Maybe we should add a date range then. The first code was written in 2022 so that's when it first applied.

Reply:
Ah I see.